pkg/service/webhook/model.go

```diff
@@ -0,0 +1,49 @@
+package webhook
+
+// In the context of webhooks, it's common to use noun.verb notation to describe events,
+// such as "credential.create" or "schema.delete".
+type Noun string
+type Verb string
+
+type Webhook struct {
+	ID   string   `json:"id" validate:"required"`
+	Noun Noun     `json:"noun" validate:"required"`
+	Verb Verb     `json:"verb" validate:"required"`
+	URLS []string `json:"urls" validate:"required"`
+}
+
+type CreateWebhookRequest struct {
+	Noun Noun     `json:"noun" validate:"required"`
+	Verb Verb     `json:"verb" validate:"required"`
+	URLS []string `json:"urls" validate:"required"`
+}
+
+type CreateWebhookResponse struct {
+	ID      string  `json:"id"`
+	Webhook Webhook `json:"webhook"`
```

Comment:
Is having ID's in both the webhook and the Create*Response intentional?

Reply:
Yup so they can know which webhook they have created and can delete.
In the next PR I will probably remove the concept of IDs alltogether and just have noun-action as the key